src/browser-realm.ts

```diff
@@ -0,0 +1,59 @@
+import { SecureEnvironment, SecureProxyTarget } from "./environment";
+import { getPrototypeOf, setPrototypeOf, getOwnPropertyDescriptors } from "./shared";
+
+const unsafeGlobalSrc = "'use strict'; this";
+
+export default function createSecureEnvironment(distortionCallback: (target: SecureProxyTarget) => SecureProxyTarget) {
+    // @ts-ignore document global ref - in browsers
+    const iframe = document.createElement('iframe');
+    iframe.style.display = 'none';
+
+    // @ts-ignore document global ref - in browsers
```

Comment:
This great. I'll do a follow-up once merged with some more iframe-y related tweaks (additive sandboxing features).